src/main/scala/com/mwz/sonar/scala/pr/GithubPrReviewJob.scala

```diff
@@ -219,6 +219,7 @@ object GithubPrReviewJob {
               }
               .groupBy { case (patchLine, _) => patchLine }
               .mapValues(_.flatMap { case (_, issuesAndComments) => issuesAndComments })
+              .toMap
```

Comment:
> However, in this case it probably would be better to use [`groupMap`](https://www.scala-lang.org/api/current/scala/collection/immutable/List.html#groupMap%5BK,B%5D(key:A=%3EK)(f:A=%3EB):scala.collection.immutable.Map%5BK,CC%5BB%5D%5D) instead.

```scala
def groupMap[K, B](key: (A) => K)(f: (A) => B): Map[K, List[B]]
```

It seems like a good idea at first, but since my `B` is a `List[(Issue, List[Comment])]` you end up with a `List[List[...]]` as the value, which you then need to flatten so you end up calling `mapValues` anyway, so you probably don't gain anything by using `groupMap` in this case.